common/src/transport_api/send.rs

```diff
@@ -0,0 +1,50 @@
+/// Implement Message trait for the type `S` with the message id `I`.
```

Comment:
I didn't rename the files themselves, just the parent. It was send because it's for messages we send IIRC.
We can tackle this when we refactor this common crate.

Reply:
Okay.